src/CognitoClient.php

```diff
@@ -239,6 +259,48 @@ public function deleteUser($accessToken)
         }
     }
 
+    /**
+     * @param string $username
+     * @param string $password
+     * @param array $userAttributes
+     * @param boolean $confirmSignup
+     * @return object
+     * @throws Exception
+     */
+    public function adminCreateUser($username, $password, $attributes = [], $confirmSignup = true)
+    {
+        try {
+            $registeredUser = $this->client->adminCreateUser([
+                'UserPoolId' => $this->userPoolId,
+                'Username' => $username,
+                'TemporaryPassword' => $password,
+                'UserAttributes' => $attributes,
+                'MessageAction' => "SUPPRESS",
+                'DesiredDeliveryMediums' => ["EMAIL"]
+            ]);
+
+            /**
+            * Auto confirm added user if confirm signup is set to true
+            */
+            if($confirmSignup){
```

Comment:
Logic like this doesn't belong in an API client, it should be the responsibility of the calling code to determine whether it needs to confirm the signup and then it should make that call separately.

Reply:
Updated this